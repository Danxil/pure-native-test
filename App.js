import React from 'react';
import {
  Platform, StyleSheet, View, StatusBar,
} from 'react-native';
import { getStatusBarHeight } from 'react-native-status-bar-height';
import AppNavigator from './components/AppNavigator';
import Provirers from './redux/Provirers';

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: '#fff',
    marginTop: getStatusBarHeight(),
  },
});

const App = () => (
  <Provirers>
    <View style={styles.container}>
      {Platform.OS === 'ios' && <StatusBar style={styles.statusBar} />}
      <AppNavigator />
    </View>
  </Provirers>
);
export default App;
