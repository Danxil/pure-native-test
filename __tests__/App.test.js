import 'react-native';
import React from 'react';
import { shallow } from 'enzyme';
import App from '../App';

it('renders correctly', () => {
  const comp = shallow(<App />);
  expect(comp).toMatchSnapshot();
});
