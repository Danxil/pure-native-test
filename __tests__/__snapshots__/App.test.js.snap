// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<pure(Providers)>
  <View
    style={
      Object {
        "backgroundColor": "#fff",
        "flex": 1,
        "marginTop": 20,
      }
    }
  >
    <StatusBar
      animated={false}
      showHideTransition="fade"
    />
    <NavigationContainer />
  </View>
</pure(Providers)>
`;
